eth/contracts/DFDiamond.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: GPL-3.0
+
+// Copy paste from SolidStateDiamond:
+// https://github.com/solidstate-network/solidstate-solidity/blob/master/contracts/proxy/diamond/SolidStateDiamond.sol
+pragma solidity ^0.8.0;
+
+import {Ownable, OwnableInternal, OwnableStorage} from "@solidstate/contracts/access/ownable/Ownable.sol";
+import {IERC173} from "@solidstate/contracts/access/IERC173.sol";
+import {ERC165, IERC165, ERC165Storage} from "@solidstate/contracts/introspection/ERC165.sol";
+import {DiamondBase, DiamondBaseStorage} from "@solidstate/contracts/proxy/diamond/base/DiamondBase.sol";
+import {DiamondReadable, IDiamondReadable} from "@solidstate/contracts/proxy/diamond/readable/DiamondReadable.sol";
+import {DiamondWritable, IDiamondWritable} from "@solidstate/contracts/proxy/diamond/writable/DiamondWritable.sol";
+
+/**
+ * @title SolidState "Diamond" proxy reference implementation
+ */
+contract DFDiamond is DiamondBase, DiamondReadable, DiamondWritable, Ownable, ERC165 {
+    using DiamondBaseStorage for DiamondBaseStorage.Layout;
+    using ERC165Storage for ERC165Storage.Layout;
+    using OwnableStorage for OwnableStorage.Layout;
+
+    constructor() {
+        ERC165Storage.Layout storage erc165 = ERC165Storage.layout();
+        bytes4[] memory selectors = new bytes4[](10);
+
+        // register DiamondWritable
+
+        selectors[0] = IDiamondWritable.diamondCut.selector;
+
+        erc165.setSupportedInterface(type(IDiamondWritable).interfaceId, true);
+
+        // register DiamondReadable
+
+        selectors[1] = IDiamondReadable.facets.selector;
+        selectors[2] = IDiamondReadable.facetFunctionSelectors.selector;
+        selectors[3] = IDiamondReadable.facetAddresses.selector;
+        selectors[4] = IDiamondReadable.facetAddress.selector;
+
+        erc165.setSupportedInterface(type(IDiamondReadable).interfaceId, true);
+
+        // register ERC165
+
+        selectors[5] = IERC165.supportsInterface.selector;
+
+        erc165.setSupportedInterface(type(IERC165).interfaceId, true);
+
+        // register Ownable
+
+        selectors[6] = Ownable.owner.selector;
+        selectors[7] = Ownable.transferOwnership.selector;
```

Comment:
Any idea why this uses `Ownable` instead of `IOwnable`? Other interfaces, like `IDiamondReadable`, are used above.

Do we want to standardize?

Reply:
```solidity
pragma solidity ^0.8.8;

import { IERC173 } from '../IERC173.sol';

interface IOwnable is IERC173 {}
```

Reply:
![Screen Shot 2022-09-22 at 11 14 29 AM](https://user-images.githubusercontent.com/42984734/191721170-26868004-e058-478f-a98a-5f895eecf137.png)

We can either use IERC173 (which is IOwnable) or leave as is, wdyt?

Reply:
![Screen Shot 2022-09-22 at 11 16 08 AM](https://user-images.githubusercontent.com/42984734/191721509-6cb1f481-8d5d-4dcf-8ef8-336dc3c4b075.png)
![Screen Shot 2022-09-22 at 11 16 23 AM](https://user-images.githubusercontent.com/42984734/191721555-7585a11e-5150-42c1-b1bb-b062f8bb26a3.png)
what do you prefer?

Reply:
I think I prefer `IERC173` because that's the actual spec we are supporting 🎉

Reply:
[27a15f7](https://github.com/dfdao/game/pull/7/commits/27a15f73805954541f49682ab51a1ae3cf9ed601)